src/plots/cartesian/tick_label_defaults.js

```diff
@@ -40,7 +40,10 @@ module.exports = function handleTickLabelDefaults(containerIn, containerOut, coe
             coerce('ticklabelstep');
         }
 
-        if(!options.noAng) coerce('tickangle');
+        if(!options.noAng) {
+            coerce('tickangle');
+            coerce('autotickangles');
```

Comment:
Lots of failures in the tests, are you able to see those? https://app.circleci.com/pipelines/github/plotly/plotly.js/9379/workflows/744eb96c-0143-4821-8862-fcec22bcb455

This logic is reused by many different axis types, so we need to exclude any that don't have `autotickangles`